app/helpers/notifications_helper.rb

```diff
@@ -230,4 +235,26 @@ def not_repo_in_active_org(param)
     return true unless param == :repo
     params[:owner].blank?
   end
+
+  def notification_status(status)
+    if status.present? && status.in?(SUBJECT_STATUS.values)
+      case status
+      when SUBJECT_STATUS[:success]
+        content_tag(:span,
+          octicon('check', height: 16, style: 'fill: white'),
+          class: 'badge badge-light',
+          style: 'background-color: #2cbe4e'
```

Comment:
I think we'll need to move the colours here into css classes so they can be easily customized for different themes.

Reply:
👍